src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```diff
@@ -757,9 +758,8 @@ public void UpdateTransitivePackageStatus(NuGetVersion installedVersion)
         {
             InstalledVersion = installedVersion;
 
-            NuGetUIThreadHelper.JoinableTaskFactory
-                .RunAsync(ReloadPackageVersionsAsync)
-                .PostOnFailure(nameof(PackageItemViewModel), nameof(ReloadPackageVersionsAsync));
+            // Transitive packages cannot be updated and can only be installed as top-level packages with their currently installed version.
```

Comment:
since this is a public method, it would be good to check if `installedVersion` argument is not null